products/sle/main.pm

```diff
@@ -613,6 +613,9 @@ sub load_inst_tests {
         elsif (get_var("LVM")) {
             loadtest "installation/partitioning_lvm";
         }
+        elsif (check_var('LVM', 0) && get_var('ENCRYPT')) {
+            loadtest 'installation/partitioning_crypt_no_lvm';
+        }
```

Comment:
can you preserve the same order as for openSUSE? I am in the process of reducing more duplicate code in both main.pm, keeping them in sync would help